diff --git a/migrations/postgresql/2020-08-02-025025_add_favorites_table/up.sql b/migrations/postgresql/2020-08-02-025025_add_favorites_table/up.sql
index 9a8787f9..dec802c4 100644
--- a/migrations/postgresql/2020-08-02-025025_add_favorites_table/up.sql
+++ b/migrations/postgresql/2020-08-02-025025_add_favorites_table/up.sql
@@ -7,2 +7,6 @@ CREATE TABLE favorites (
 
+COMMIT;
+
+BEGIN;
+
 -- Transfer favorite status for user-owned ciphers.
