diff --git a/change_color.sh b/change_color.sh
index 8d353212..4e7510c2 100755
--- a/change_color.sh
+++ b/change_color.sh
@@ -153,13 +153,6 @@ if [[ "$SRC_PATH" == "$DEST_PATH" ]]; then
 fi
 
 
-tempdir=$(mktemp -d)
-post_clean_up() {
-  rm -r "$tempdir" || :
-}
-trap post_clean_up EXIT SIGHUP SIGINT SIGTERM
-cp -r "$SRC_PATH/"* "$tempdir/"
-cd "$tempdir"
 
 
 # autodetection which color variant to use
@@ -300,17 +293,6 @@ fi
 echo "== Rendering GTK 3 assets..."
 FORCE_INKSCAPE="$OPTION_FORCE_INKSCAPE" ./render-assets.sh gtk
 
-FORCE_INKSCAPE="$OPTION_FORCE_INKSCAPE" ./install.sh --dest "$TARGET_DIR" --name "${OUTPUT_THEME_NAME/\//-}" --color "$COLOR_VARIANT" --size "$SIZE_VARIANT"
-
-GENERATED_PATH="$DEST_PATH$(tr -d ',' <<< "$COLOR_VARIANTS")$(tr -d ',' <<< "$SIZE_VARIANTS")"
-if [[ "$GENERATED_PATH" != "$DEST_PATH" ]]; then
-  if [[ -d "$DEST_PATH" ]]; then
-    rm -r "$DEST_PATH"
-  fi
-  mv "$GENERATED_PATH" "$DEST_PATH"
-fi
-
 echo
 echo "== SUCCESS"
-echo "== The theme was installed to '$DEST_PATH'"
 exit 0
