diff --git a/internal/database/migrations.go b/internal/database/migrations.go
index e42e329a..17e0cfa0 100644
--- a/internal/database/migrations.go
+++ b/internal/database/migrations.go
@@ -356,3 +356,3 @@ var migrations = []func(tx *sql.Tx) error{
 	func(tx *sql.Tx) (err error) {
-		sql := `CREATE INDEX enclosures_user_entry_url_idx ON enclosures(user_id, entry_id, md5(url))`
+		sql := `CREATE INDEX enclosures_user_entry_url_idx ON enclosures(user_id, entry_id, url);`
 		_, err = tx.Exec(sql)
@@ -465,3 +465,3 @@ var migrations = []func(tx *sql.Tx) error{
 		_, err = tx.Exec(`
-			CREATE INDEX entries_feed_url_idx ON entries(feed_id, url) WHERE length(url) < 2000;
+			CREATE INDEX entries_feed_url_idx ON entries(feed_id, url);
 			CREATE INDEX entries_user_status_feed_idx ON entries(user_id, status, feed_id);
@@ -692,3 +692,3 @@ var migrations = []func(tx *sql.Tx) error{
 		// Create unique index
-		_, err = tx.Exec(`CREATE UNIQUE INDEX enclosures_user_entry_url_unique_idx ON enclosures(user_id, entry_id, md5(url))`)
+		_, err = tx.Exec(`CREATE UNIQUE INDEX enclosures_user_entry_url_unique_idx ON enclosures(user_id, entry_id, url);`)
 		if err != nil {
diff --git a/internal/storage/enclosure.go b/internal/storage/enclosure.go
index ba0a69de..01f9fec4 100644
--- a/internal/storage/enclosure.go
+++ b/internal/storage/enclosure.go
@@ -157,3 +157,3 @@ func (s *Storage) createEnclosure(tx *sql.Tx, enclosure *model.Enclosure) error
 			($1, $2, $3, $4, $5, $6)
-		ON CONFLICT (user_id, entry_id, md5(url)) DO NOTHING
+		ON CONFLICT (user_id, entry_id, url) DO NOTHING
 		RETURNING
