diff --git a/internal/database/migrations.go b/internal/database/migrations.go
index f835859d..698918cc 100644
--- a/internal/database/migrations.go
+++ b/internal/database/migrations.go
@@ -279,3 +279,3 @@ var migrations = []func(tx *sql.Tx) error{
 		sql := `
-			UPDATE entries SET document_vectors = to_tsvector(substring(title || ' ' || coalesce(content, '') for 1000000));
+			UPDATE entries SET document_vectors = to_tsvector(substring(title || ' ' || title || ' ' || coalesce(content, '') for 1000000));
 			CREATE INDEX document_vectors_idx ON entries USING gin(document_vectors);
@@ -290,12 +290,2 @@ var migrations = []func(tx *sql.Tx) error{
 	},
-	func(tx *sql.Tx) (err error) {
-		sql := `
-			UPDATE
-				entries
-			SET
-				document_vectors = setweight(to_tsvector(substring(coalesce(title, '') for 1000000)), 'A') || setweight(to_tsvector(substring(coalesce(content, '') for 1000000)), 'B')
-		`
-		_, err = tx.Exec(sql)
-		return err
-	},
 	func(tx *sql.Tx) (err error) {
diff --git a/internal/storage/entry.go b/internal/storage/entry.go
index 272e11d2..f37422d5 100644
--- a/internal/storage/entry.go
+++ b/internal/storage/entry.go
@@ -79,3 +79,3 @@ func (s *Storage) UpdateEntryTitleAndContent(entry *model.Entry) error {
 			reading_time=$3,
-			document_vectors = setweight(to_tsvector($4), 'A') || setweight(to_tsvector($5), 'B')
+			document_vectors = to_tsvector(substring($4 || ' ' || $4 || ' ' || coalesce($5, '') for 1000000))
 		WHERE
@@ -132,3 +132,3 @@ func (s *Storage) createEntry(tx *sql.Tx, entry *model.Entry) error {
 				now(),
-				setweight(to_tsvector($11), 'A') || setweight(to_tsvector($12), 'B'),
+				to_tsvector(substring($11 || ' ' || $11 || ' ' || coalesce($12, '') for 1000000)),
 				$13
@@ -190,3 +190,3 @@ func (s *Storage) updateEntry(tx *sql.Tx, entry *model.Entry) error {
 			reading_time=$6,
-			document_vectors = setweight(to_tsvector($7), 'A') || setweight(to_tsvector($8), 'B'),
+			document_vectors =  to_tsvector(substring($7 || ' ' || $7 || ' ' || coalesce($8, '') for 1000000)),
 			tags=$12
